src/middlewared/middlewared/plugins/network.py

```diff
@@ -1110,9 +1110,12 @@ async def _common_validation(self, verrors, schema_name, data, itype, update=Non
             if data['lag_protocol'] not in await self.middleware.call('interface.lag_supported_protocols'):
                 verrors.add(
                     f'{schema_name}.lag_protocol',
-                    f'{platform.system()} does not support LAGG protocol {data["lag_protocol"]}',
+                    f'{platform.system()} does not support LAG protocol {data["lag_protocol"]}',
```

Comment:
if `data["lag_protocol"]` is empty, error message will be `"FreeBSD does not support LAG protocol "`

Reply:
Can it be empty if we have enum in the schema?

Reply:
Good point, sorry